engines/experiment/plugin/rpc/factory.go

```diff
@@ -3,16 +3,24 @@ package rpc
 import (
 	"encoding/json"
 	"fmt"
+	"net/url"
+	"path"
 	"reflect"
 	"sync"
 
+	"github.com/gojek/turing/engines/experiment/config"
 	"github.com/gojek/turing/engines/experiment/manager"
+	"github.com/gojek/turing/engines/experiment/pkg/utils"
 	"github.com/gojek/turing/engines/experiment/plugin/rpc/shared"
 	"github.com/gojek/turing/engines/experiment/runner"
 	"github.com/hashicorp/go-plugin"
+	"github.com/mitchellh/hashstructure/v2"
```

Comment:
Probably insignificant question, but is there a reason we're using this package to generate hashes over others?

Reply:
No strong preferences, but we are already using https://github.com/mitchellh/mapstructure hence I didn't think too long before bringing this package in as well